blocks/newslist/newslist.css

```diff
@@ -36,21 +36,45 @@
   margin-right: auto;
 }
 
+@media (min-width: 1000px) {
```

Comment:
please move this style to where we've all the desktop styles in https://github.com/hlxsites/accenture-newsroom/blob/ed03a0b640d794db5dd62056e3fe6057c703a6bf/blocks/newslist/newslist.css#L383

Reply:
@ravkiran done moving the css styling.